### ISSUE TITLE
mobile menu icon fa-bars fails passing achecker accessibility review AA

### ISSUE THREAD

**Issue by szoupi** [gantry/gantry5#3054]
The file media/gantry5/engines/nucleus/templates/page.html.twig contains the code (line 91)
<i class="fa fa-fw fa-bars">
which fails the Success Criteria 1.4.4 Resize text (AA) when checking for accessibility (https://achecker.achecks.ca/checker/index.php)
It can be manually overridden on templates (engine/templates/page.html.twig) changing <i> with <span> but a permanent solution will be much better.

![image](https://user-images.githubusercontent.com/12526216/175781050-9e7d24ab-cefe-4f92-be28-248ecb1f2e5e.png)

**Comment by N8Solutions** [comment]
@szoupi It's good you are looking into these things but thanks to @mahagr & @w00fz this was handled using JS just over 5 years ago, quite literally, and I was the one that opened the ticket for it back then. 
You can check it out here to better understand how it handles the accessibility aspect: https://github.com/gantry/gantry5/issues/1891

Another important thing is that the accessibility checker you are using is not inspecting the websites as they would be seen on a mobile device. They are only inspecting it as a desktop version. Along with that, you need to keep in mind that it is simply searching for specific code, and reporting what it finds, which still requires human interpretation.

In version 5 of the Font Awesome [documentation](https://fontawesome.com/v5/docs/web/reference-icons/) it says this:

> Font Awesome is designed to be used with inline elements, and we recommend sticking with a consistent HTML element to reference them by in your project. We like the `<i>` tag for brevity and because most folks use `<em></em>` for emphasized/italicized semantic text these days. If that's not your cup of tea, using a `<span>` is more semantically correct.

You can read further about properly using the FA icons for accessibility for **decorative icons** and **semantic icons** starting [here](https://fontawesome.com/v5/docs/web/other-topics/accessibility#web-fonts-with-css-decorative-icons) which incorporates the use of the _aria-hidden_ attribute.

I hope my explanation helps you understand what is going on. Feel free to ask any questions.

When you get a chance, please close this issue.

**Comment by szoupi** [comment]
Got it, thanks!
